server/src/controllers/christmas-tree.es6

```diff
@@ -500,6 +483,25 @@ christmasTree.printPermit = (req, res) => {
     });
 };
 
+const completePermitTransaction = (permit, res, req) => {
+  util.logControllerAction(req, 'christmasTree.completePermitTransaction', permit);
+  const xmlData = paygov.getXmlToCompleteTransaction(permit.paygovToken);
+  paygov.postPayGov(xmlData)
```

Comment:
This function is not returning the promise and i notice that the caller (`updatePermitApplication`) is not expecting it to. Wondering if it would make sense to return it just in case...